StormCPackConfig.cmake

```diff
@@ -1,31 +1,21 @@
 include(InstallRequiredSystemLibraries)
 
-# For help take a look at:
-# http://www.cmake.org/Wiki/CMake:CPackConfiguration
-
-### general settings
-set(CPACK_PACKAGE_NAME "Storm")
+### General settings
+# General project information is taken from the project() variables
 set(CPACK_PACKAGE_VENDOR "Storm Developers")
-set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Storm - A Modern Probabilistic Model Checker.")
 set(CPACK_PACKAGE_CONTACT "support@stormchecker.org")
-
 set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_SOURCE_DIR}/LICENSE")
-
-### versions
-set(CPACK_PACKAGE_VERSION_MAJOR "${STORM_VERSION_MAJOR}")
-set(CPACK_PACKAGE_VERSION_MINOR "${STORM_VERSION_MINOR}")
-set(CPACK_PACKAGE_VERSION_PATCH "${STORM_VERSION_PATCH}")
-
-set(CPACK_GENERATOR "ZIP")
+set(CPACK_RESOURCE_FILE_README "${CMAKE_SOURCE_DIR}/README.md")
 set(CPACK_PACKAGE_INSTALL_DIRECTORY "${CPACK_PACKAGE_NAME}-${CPACK_PACKAGE_VERSION}")
 
-### source package settings
-set(CPACK_SOURCE_GENERATOR "ZIP")
+set(CPACK_GENERATOR "DEB")
+
+### Source package settings
+set(CPACK_SOURCE_GENERATOR "TGZ")
```

Comment:
The source generator is no longer used? But then we should probably clear what follows here and below?

Reply:
Do we want to have the possibility to create a source package? Or is it sufficient that any Github release already comes with an archive of the sources? I think it should be fine to remove the support for a source package here.